pom.xml

```diff
@@ -147,6 +153,17 @@
         <artifactId>httpcore</artifactId>
         <version>${version.httpcomponents.httpcore}</version>
       </dependency>
+      <dependency>
+        <groupId>org.slf4j</groupId>
+        <artifactId>slf4j-api</artifactId>
+        <version>1.7.26</version>
+      </dependency>
+      <dependency>
+        <groupId>ch.qos.logback</groupId>
+        <artifactId>logback-classic</artifactId>
+        <version>1.2.3</version>
+        <scope>test</scope>
```

Comment:
Please do not use `<scope>` under dependencyManagement. Having it there is not reliable and thus redundant.